uaclient/contract.py

```diff
@@ -177,6 +177,30 @@ def report_machine_activity(self):
             machine_token["activityInfo"] = response
             self.cfg.write_cache("machine-token", machine_token)
 
+    def get_updated_contract_info(
```

Comment:
bikeshed naming suggestion
```suggestion
    def get_contract_info(
```

Reply:
There is another [function ](https://github.com/canonical/ubuntu-advantage-client/blob/25c99452a7ec4014cbd7e36f23653e2f3eb664b7/uaclient/contract.py#L565) with the name `get_contract_information` which I think will be having the same function name, technically.